proj/src/main/webapp/script.js

```diff
@@ -429,10 +433,10 @@ function initUserChat(){
 
 }
 // Sets title of page
-function setTitle(){
-    nameRef = firebase.database().ref("/chat/"+tag+"/"+globalChatId+"/chatInfo/name");
+function setTitle(dbRefObj){
+    nameRef = dbRefObj.parent.child('chatInfo');
```

Comment:
Before anyone comments on this, the dbRefObj was the wrong child of where I needed, but this lets me reuse that object